tests/openwrt/test_interfaces.py

```diff
@@ -197,6 +213,17 @@ def test_parse_missing_ipv4_mask_list(self):
         o = OpenWrt(native=native)
         self.assertEqual(o.config, self._eth0_32)
 
+    def test_parse_missing_ipv6_mask_list(self):
+        native = self._tabs("""package network
+
+config interface 'eth0'
+    option ifname 'eth0'
+    list ipaddr 'fd87::1'
```

Comment:
on OpenWRT, `ipaddr` is for ipv4, you have to use `ip6addr` to trigger your test, use the following:

```
option ip6addr '2aa1:4aaa:2aaa:1d::5/64'
```

Update the expected value accordingly as well.